### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/barcode.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "milon/barcode": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: composer.json [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
+    "license": "LGPL-3.0",
+    "authors": [
+        {
+            "name": "Nuruzzaman Milon",
+            "email": "contact@milon.im"
+        }
+    ],
+    "require": {
+        "php": "^7.3 | ^8.0",
+        "illuminate/support": "^9.0"
```
Comment (xorock) [inline composer.json:25]: To keep backwards compatibility:
`"illuminate/support": "^7.0|^8.0|^9.0"

**Comment by PaolaRuby** [comment]
Closes #144

**Comment by erikn69** [comment]
@milon please review 👀

**Comment by milon** [comment]
thanks

**Comment by jasonmccreary** [comment]
@milon very welcome!

**Pull request merged**